tools/update_extern.py

```diff
@@ -0,0 +1,166 @@
+#!/bin/env python
+
+"""
+This script is used to update the files in `sunpy/extern` while making new releases.
+"""
+
+import os
+import shutil
+from zipfile import ZipFile
+import requests
+from pathlib import Path
+
+# get parent directory of sunpy
+SUNPY_DIR = Path(__file__).parent.parent
+
+
+# Dictionary to store the authors of the packages
+PACKAGES = {
+    "appdirs": "ActiveState",
+    "distro": "python-distro",
+    "inflect": "jaraco",
+    "parse": "r1chardj0n3s",
+}
+
+
+def package_exists():
+    """
+    Check if the package exists.
+    """
+    # This function checks if AUTHORS and PACKAGES are correctly match
+    for package in PACKAGES:
+        print(f"Checking {PACKAGES[package]}/{package}")
+        # Get 200 response from github
+        response = requests.get(f"https://api.github.com/repos/{PACKAGES[package]}/{package}")
+        if response.status_code != 200:
+            print(f"{PACKAGES[package]}/{package} does not exist.")
+            return False
+    return True
+
+
+def get_latest_version(author, package):
+    """
+    Get the latest version of the package.
+    """
+    url = f"https://api.github.com/repos/{author}/{package}/releases/latest"
+    response = requests.get(url)
+    return response.json()["tag_name"]
+
+
+def get_download_url(author, package, version):
+    """
+    Get the download url for the package.
+    """
+    url = f"https://github.com/{author}/{package}/archive/refs/tags/{version}.zip"
+    return url
```

Comment:
This seems like it can be one function or one block of code?

Reply:
Yes, I will do that.